code/game/gamemodes/wizard/artefact.dm

```diff
@@ -283,8 +283,8 @@ var/global/list/multiverse = list()
 	if(prob(50))
 		var/list/all_species = list()
 		for(var/speciestype in subtypesof(/datum/species))
-			var/datum/species/S = new speciestype()
-			if(!S.dangerous_existence)
+			var/datum/species/S = speciestype
+			if(!initial(S.dangerous_existence))
 				all_species += speciestype

```

Comment:
uh, no, it really wasn't @ChangelingRain 
its used PURELY for the dangerous existence check.